runtime/zeitgeist/src/parachain_params.rs

```diff
@@ -48,11 +48,12 @@ parameter_types! {
     // Cumulus and Polkadot
     pub Ancestry: MultiLocation = Parachain(ParachainInfo::parachain_id().into()).into();
     pub const RelayLocation: MultiLocation = MultiLocation::parent();
-    pub const RelayNetwork: NetworkId = NetworkId::Kusama;
-    pub const ReservedDmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
-    pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
+    // Have to change "Any" to "Kusama" for mainnet once we have separate runtimes
+    pub const RelayNetwork: NetworkId = NetworkId::Any;
+    pub const ReservedDmpWeight: Weight = Weight::from_ref_time(MAXIMUM_BLOCK_WEIGHT.ref_time() / 4);
+    pub const ReservedXcmpWeight: Weight = Weight::from_ref_time(MAXIMUM_BLOCK_WEIGHT.ref_time() / 4);
```

Comment:
Would prefer this way.

https://github.com/PureStake/moonbeam/blob/2c7c152cc805326014f6989f8502f66952d33731/runtime/moonbase/src/lib.rs#L619-L620

But your choice.